src/ray/gcs/gcs_server/gcs_server_main.cc

```diff
@@ -120,7 +148,7 @@ int main(int argc, char *argv[]) {
   gcs_server_config.redis_username = redis_username;
   gcs_server_config.retry_redis = retry_redis;
   gcs_server_config.node_ip_address = node_ip_address;
-  gcs_server_config.log_dir = log_dir;
+  gcs_server_config.log_dir = event_log_dir;
```

Comment:
Oh, this means this is not only used to store events:

```
void GcsServer::DumpDebugStateToFile() const {
  std::fstream fs;
  fs.open(config_.log_dir + "/debug_state_gcs.txt",
          std::fstream::out | std::fstream::trunc);
  fs << GetDebugState() << "\n\n";
  fs << io_context_provider_.GetDefaultIOContext().stats().StatsString();
  fs.close();
}
```

In this case, it's still better to name the option as `--log-dir` instead of `--event-log-dir`.

Reply:
Ok, updated.